File: Assignments/Assignment_18/p2.c
#include<stdio.h>
void main(){

        int row,count =1;
        printf("Enter the no of rows :\n");
        scanf("%d",&row);
        for(int i=row;i>0;i--){

                for(int j=i;j>0;j--){

                        printf("%d\t",count);
                }printf("\n\n");
                count++;
        }
}

File: JAVA/9.Nested_For_2/p2.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		for(int i=0;i<row;i++){
		
			char ch=(char)(row+96);
			for(int j=0;j<row;j++){
			
				System.out.print((ch--)+"\t");
			}System.out.println();
		}
	}
}
